src/Sentry/SentryOptions.cs

```diff
@@ -77,6 +77,20 @@ public bool IsGlobalModeEnabled
         set => _isGlobalModeEnabled = value;
     }
 #endif
+    /// <summary>
+    /// Strategy used by the handler
+    /// </summary>
+    /// <code>
+    /// SentrySdk.Init(options =>
+    /// {
+    /// Sentry
+    /// // ...
+    /// options.HandlerStrategy = NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_DEFAULT;
+    /// // ... other options
+    /// }
+    /// </code>
+    /// <seealso href="https://github.com/dotnet/android/issues/9055#issuecomment-2261347912"/>
+    public NdkHandlerStrategy HandlerStrategy { get; set; } = NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_DEFAULT;
```

Comment:
Yes, now that you mention it, I think it would make more sense to have `SENTRY_HANDLER_STRATEGY_CHAIN_AT_START ` as a default as a constant, as I don't see any scenario where we wouldn't use it. Especially since the issue comes from dotnet-generated machine code